float32 rad
float32 azi
float32 ele
float32 temperature
int32 index